explainer.md

```diff
@@ -580,6 +618,11 @@ enum XRSessionMode {
   "immersive-ar"
 }
 
+dictionary XRSessionInit {
```

Comment:
This still needs to be added to the requestSession argument list above?

```webidl
Promise<XRSession> requestSession(XRSessionMode mode, optional XRSessionInit options);
```

Speaking of this, is the intent that these init options only apply to `requestSession`, not `supportsSession`? If yes, I think it would be helpful to call that out specifically in the explainer. For example, asking for options may require prompting the user to get an answer, and `supportsSession` is intended to be non-interactive.

Reply:
Yes, the intent is these options only apply to `requestSession`. Not only would it violate the desire for `supportsSession` to be non-interactive to properly resolve some of the requirements, it would also make it a much more potent fingerprinting venue. As such `supportsSession` will remain `mode`-only.

Reply:
Thanks for catching that!